### PR TITLE
Add links to generate a mail template to subscribe and unsubscribe for more convinience

### PR DESCRIPTION
**Pull request by daipom**
Fix mail addresses into mailto tags so that we can subscribe the address
just by clicking the link and sending the mail, which is automatically created.

I haven't finished this yet.

I will run the rakefile, and check if this is working properly.
I need to add the diff of `index.html` after running the rakefile.

### CODE REVIEW

**Comment by piroor** [comment]
差分の方は特に問題はないように感じましたが、コミットメッセージ（とプルリクエストのタイトル）としては
Generate links to start message composition with template to subscribe for more convenience
とかの方が説明的でよいのではないだろうか？と思いました。
また、リンクのラベルをSubscribeとして、同様にUnsubscribeというラベルでunsubscribe用のリンクも用意するといいのではないかと思いました。
いかがでしょう？

File: Rakefile [hunk 1]
```diff
@@ -80,7 +80,7 @@ class RepositoryListUpdater
             <tr>
               <td><a href="#{owner_url(fqdn, owner)}">#{h(owner)}</a></td>
               <td>#{repository_column}</td>
-              <td>#{h(to)}</td>
+              <td><a href="mailto:#{h(to)}?cc=null@commit-email.info&amp;subject=Subscribe&amp;body=subscribe">#{h(to)}</a></td>
```
Comment (kou) [inline Rakefile]: Could you extract this `mailto:` URL generate code as a method like `owner_url` and `repository_url`?

**Comment by daipom** [comment]
@piroor ありがとうございます。
メッセージも、リンクのことも、その方が分かりやすくて良いと思います。
タイトル変更して、コードも修正してみます。

File: Rakefile (hunk 1 above)
Comment (daipom) [reply Rakefile]: Thank you for your advise.
It will make the code very readable.
I will fix it.

Comment (daipom) [reply Rakefile]: I have fixed this.

**Comment by daipom** [comment]
I have added `Subscribe` and `Unsubscribe` columns to `Repositories` section.
We can get the mail template by clicking these buttons and can send it as it is to subscribe or subscribe.

I have also added some texts explaining this to `How to use` section.

File: Rakefile [hunk 2]
```diff
@@ -92,6 +96,14 @@ class RepositoryListUpdater
   def repository_url(fqdn, owner, repository)
     "#{owner_url(fqdn, owner)}#{h(repository)}/"
   end
+
+  def mailto_href_subscribe(to)
```
Comment (kou) [inline Rakefile]: `href` is for `<a>`. This is used for `href` but this is an URL not `href`.

```suggestion
  def subscribe_url(to)
```

(I think that `mailto` is redundant here. Should we include it in method name?)

File: ansible/files/var/www/html/index.html [hunk 3]
```diff
@@ -44,6 +44,8 @@ <h3>How to subscribe</h3>
 --
 subscribe
         </pre>
+
+        <p>* You can get the mail template by clicking the "Subscribe" button in the list and can send it as it is.</p>
```
Comment (kou) [inline ansible/files/var/www/html/index.html]:
```suggestion
        <p>* You can get the mail template by clicking the "Subscribe" link in the list and can send it as it is.</p>
```

File: Rakefile [hunk 4]
```diff
@@ -92,6 +96,14 @@ class RepositoryListUpdater
...
+    "mailto:#{h(to)}?cc=null@commit-email.info&amp;subject=Subscribe&amp;body=subscribe"
```
Comment (kou) [inline Rakefile]: Ah, we should call `h` when we use an URL not generate an URL. `&amp;` is for HTML not for URL.
I've updated the current implementation. Please follow the convention.

File: Rakefile (hunk 2 above)
Comment (daipom) [reply Rakefile]: Thank you for the advice.
That's exactly right.
I have fixed this.

File: ansible/files/var/www/html/index.html (hunk 3 above)
Comment (daipom) [reply ansible/files/var/www/html/index.html]: Thank you for the suggestion.
I have fixed this.

File: Rakefile (hunk 4 above)
Comment (daipom) [reply Rakefile]: Thank you for the advice.
We should enclose the whole URL in `h` and leave the conversion to `&amp;` to `h`, right?

```rb
h("mailto:#{to}?cc=null@commit-email.info&subject=Subscribe&body=subscribe")
```

Comment (daipom) [reply Rakefile]: Oh, I should refer to this fix. Thank you.
https://github.com/kou/commit-email.info/commit/38c11ffc8f244e921c52a1ad0320693094210294

Comment (kou) [reply Rakefile]: This method should not call `h`.
See also: https://github.com/kou/commit-email.info/commit/38c11ffc8f244e921c52a1ad0320693094210294

Comment (daipom) [reply Rakefile]: Thank you. I understand.
`h` must be called on href directly.
I have fixed this.

**Comment by daipom** [comment]
I haven't been able to confirm the behavior with vagrant,
but I have confirmed that opening the html file in a browser and clicking on the link will open the intended email.

**Comment by kou** [comment]
Thanks!

**Comment by daipom** [comment]
@kou @piroor 
Thank you very much for reviewing and merging!

**Pull request merged**